Add docs for 32bits global kill (en) (#14405)
pingcap/docs: tidb-configuration-file.md

+ Controls whether to enable the Global Kill (terminating queries or connections across instances) feature.
+ Default value: `true`
+ When the value is `true`, both `KILL` and `KILL TIDB` statements can terminate queries or connections across instances so you do not need to worry about erroneously terminating queries or connections. When you use a client to connect to any TiDB instance and execute the `KILL` or `KILL TIDB` statement, the statement will be forwarded to the target TiDB instance. If there is a proxy between the client and the TiDB cluster, the `KILL` and `KILL TIDB` statements will also be forwarded to the target TiDB instance for execution.

> + Since v7.3.0, using the MySQL command line <kbd>Control+C</kbd> to terminate a query or connection in TiDB is supported when both `enable-global-kill` and `enable-32bits-connection-id` are `true`. For more information on the `KILL` statement, see [KILL](/sql-statements/sql-statement-kill.md).

Reviewer: ```suggestion
+ Starting from v7.3.0, you can terminate a query or connection using the MySQL command line <kbd>Control+C</kbd> when `enable-global-kill = true` and [`enable-32bits-connection-id = true`](#enable-32bits-connection-id-new-in-v730). For more information, see [`KILL`](/sql-statements/sql-statement-kill.md).
```

> + Starting from v7.3.0, you can terminate a query or connection using the MySQL command line <kbd>Control+C</kbd> when `enable-global-kill = true` and [`enable-32bits-connection-id = true`](#enable-32bits-connection-id-new-in-v730). For more information, see [`KILL`](/sql-statements/sql-statement-kill.md).

Reviewer 2: ```suggestion
+ Starting from v7.3.0, you can terminate a query or connection using the MySQL command line <kbd>Control+C</kbd> when both `enable-global-kill` and [`enable-32bits-connection-id`](#enable-32bits-connection-id-new-in-v730) are set to `true`. For more information, see [`KILL`](/sql-statements/sql-statement-kill.md).
```

### `enable-32bits-connection-id` <span class="version-mark">New in v7.3.0</span>

+ Controls whether to enable the 32-bit connection ID feature.
+ Default value: `true`

> + When this configuration item, as well as `enable-global-kill`, is set to true, 32-bit connection IDs will be generated, allowing the termination of queries or connections by the MySQL command-line <kbd>Control+C</kbd>.

Reviewer: ```suggestion
+ When both this configuration item and [`enable-global-kill`](#enable-global-kill-new-in-v610) are set to `true`, TiDB generates 32-bit connection IDs. This enables you to terminate queries or connections by the MySQL command-line <kbd>Control+C</kbd>.
```

> **Warning:**
>

> > When the number of TiDB instances in the cluster exceeds 2048 or the concurrent connection count of a single TiDB instance exceeds 1048576, the 32-bit connection ID space is insufficient and will be automatically upgraded to 64-bit connection ID. During the upgrade process, existing business and established connections will not be affected. However, subsequent new connections will not be able to be terminated using <kbd>Control+C</kbd> in the MySQL command-line.

Reviewer: ```suggestion
> When the number of TiDB instances in the cluster exceeds 2048 or the concurrent connection count of a single TiDB instance exceeds 1048576, the 32-bit connection ID space becomes insufficient and is automatically upgraded to 64-bit connection IDs. During the upgrade process, existing business and established connections are unaffected. However, subsequent new connections cannot be terminated using <kbd>Control+C</kbd> in the MySQL command-line.
```
